torch/multiprocessing/spawn.py

```diff
@@ -18,8 +18,20 @@ def _wrap(fn, i, args, error_queue):
         sys.exit(1)
 
 
+def _python_version_check():
+    if sys.version_info < (3, 4):
+        raise RuntimeError("Requires python 3.4 or higher to use "
+                           "torch.multiprocessing.spawn and "
+                           "torch.multiprocessing.SpawnContext helper "
```

Comment:
No need to refer to spawn context, people shouldn't use it. Only reason for exposing it is that it is picked up by the docs.

Reply:
Well, if we expose it, we can't guarantee it will not be used by people. That's why I was asking if we should hide it offline